### DISCUSSION TITLE
4ChPro R3 - RF433 Learning mode?

### DISCUSSION THREAD

**Discussion by eric-joris** [arendst/Tasmota#11560] [Q&A] [answered]
I just flashed a 4CHPRO R3 with Tasmota 9.3.1 firmware (tasmota.bin). All seems to work fine. I know like to control the 4CHPRO with the RF remote. How can I pair the RF433 remote with the 4CHPRO? The Sonoff methode does not work. Thanks, Eric

**Comment by TrDA-hab** [comment, marked as answer]
RF433 - not tied to Tasmota in any way. Only Sonoff's method should (!) Work.

**Reply by eric-joris** [reply]
This confirms what I expected. I will try again using the Sonoff method. Many thanks for your reply. Eric

**Reply by TrDA-hab** [reply]
98% of these are problems with the remote control: incompatibility with Sonoff or discharged battery.
